spec/controllers/step_by_step_pages_controller_spec.rb

```diff
@@ -90,12 +90,35 @@
     end
   end
 
-  describe "#unschedule" do
+  describe "#schedule" do
+    let(:step_by_step_page) { create(:draft_step_by_step_page) }
+
     before :each do
-      stub_user.permissions << "Scheduling"
+      stub_scheduling_permissions
+      stub_publishing_api_for_scheduling
+    end
 
-      allow(Services.publishing_api).to receive(:get_content)
-        .and_return(content_item(step_by_step_page))
+    it "sets `scheduled_at` to a datetime" do
+      expect(step_by_step_page.scheduled_at.class.name).to eq 'NilClass'
+
+      schedule_publishing(step_by_step_page)
+
+      expect(step_by_step_page.scheduled_at.class.name).to eq 'Time'
```

Comment:
Rather than using a class name, it would be better to check the actual value being stored, so that when the inputs for `scheduled_at` are separated into date and time elements later, we can check that we aren't losing any data, e.g. dropping the number of minutes of storing the "am" time rather than the "pm" etc.

Reply:
Done 👍